web/analytics/models.py

```
@@ -0,0 +1,109 @@
+from django.db import models
+from django.contrib.auth.models import User
+from web.models import Course, Session, Enrollment, CourseProgress
```

Ruff (0.8.2)

`web.models.Session` imported but unused

Remove unused import

(F401)